src/java.base/share/classes/java/util/concurrent/atomic/AtomicReference.java

```diff
@@ -353,8 +353,8 @@ public final void setRelease(V newValue) {
      *
      * @param expectedValue the expected value
      * @param newValue the new value
-     * @return the witness value, which will be the same as the
-     * expected value if successful
+     * @return the <em>witness value</em>, which is the current value
+     * at the time of the operation.
```

Comment:
This change is not appropriate as discussed in the PR. And unclear why you changed only this version? Leftover?

Reply:
@dholmes-ora Good catch, will fix.